README.md

```diff
@@ -118,6 +118,22 @@ make freeze-requirements
 
 `requirements.txt` should be committed alongside `requirements-app.txt` changes.
 
+## Using Local Jinja for template changes
```

Comment:
Maybe 

## Using Local Jinja for testing template changes

Just want to be 100% clear this is for local testing only